### ISSUE TITLE
Error: obj.hasOwnProperty is not a function

### ISSUE THREAD

**Issue by wfel** [mzabriskie/axios#436]
I'm using axios with vue-router and I'm doing request with query like this:
```javascript
axios.get(
        '/search',
        {params: this.$route.query}
    );
```
and i got this error.
The problem is that `this.$route.query` is an object created with `Object.create(null)` and it does not have `hasOwnProperty` on its prototype chain. I wonder if you will fix it or leave it as standard behavior.

**Comment by tony-kerz** [comment]
yes, i just got burned by this as well using from node, something like:
```
  import axios from 'axios'
  import queryString from 'querystring'

  const query='foo=bar'
  const params = queryString.parse(query)
  return axios.get(url, {params})
```

**Comment by mzabriskie** [comment]
This should be fixed once #445 is released.
